DSharpPlus/Entities/DiscordWidgetSettings.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities
+{
+	/// <summary>
+	/// Represents a Discord guild's widget settings.
+	/// </summary>
+	public class DiscordWidgetSettings
+    {
+        internal DiscordGuild Guild { get; set; }
+
+        [JsonProperty("channel_id", NullValueHandling = NullValueHandling.Ignore)]
+        internal ulong ChannelId { get; set; }
+
+        [JsonProperty("enabled", NullValueHandling = NullValueHandling.Ignore)]
+        internal bool Enabled { get; set; }
+
+        /// <summary>
+        /// Gets the guild's widget channel.
+        /// </summary>
+        public DiscordChannel Channel
+            => this.Guild.GetChannel(this.ChannelId);
+
+        /// <summary>
+        /// Gets if the guild's widget is enabled.
+        /// </summary>
+        public bool IsEnabled
```

Comment:
Can't you just directly expose Enabled?

Reply:
I thought about it, but I went with IsEnabled to match other bool properties in the library such as IsBot, IsOwner, etc.

Happy to change it if you prefer, however.

Reply:
```cs
/// <summary>
/// Gets whether the user is a bot.
/// </summary>
[JsonProperty("bot", NullValueHandling = NullValueHandling.Ignore)]
public virtual bool IsBot { get; internal set; }
```
`IsBot` in `DiscordUser` is deserialized directly from `bot` json property. So it is fine. `IsOwner` is actually the result of a comparison between current user id and `OwnerId`. So directly exposing the field (and renaming it `IsEnabled`) would be just fine! 👍